File: src/main/java/com/sandjelkovic/dispatchd/domain/data/entity/ImportProgressStatus.java
package com.sandjelkovic.dispatchd.domain.data.entity;

public enum ImportProgressStatus {
	QUEUED("QUEUED"),
	IN_PROGRESS("IN_PROGRESS"),
	SUCCESS("SUCCESS"),
	SHOW_ALREADY_EXISTS("SHOW_ALREADY_EXISTS"),
	REMOTE_SERVER_ERROR("REMOTE_SERVER_ERROR"),
	ERROR("ERROR");

	private final String field;

	ImportProgressStatus(String field) {
		this.field = field;
	}

	@Override
	public String toString() {
		return field;
	}
}

File: content-service/src/main/kotlin/com/sandjelkovic/dispatchd/contentservice/data/entity/ImportProgressStatus.kt
package com.sandjelkovic.dispatchd.contentservice.data.entity

/**
 * @author sandjelkovic
 * @date 21.4.18.
 */
enum class ImportProgressStatus(val field: String) {
    QUEUED("QUEUED"),
    IN_PROGRESS("IN_PROGRESS"),
    SUCCESS("SUCCESS"),
    ERROR_SHOW_ALREADY_EXISTS("ERROR_SHOW_ALREADY_EXISTS"),
    ERROR_REMOTE_SERVER("ERROR_REMOTE_SERVER"),
    ERROR("ERROR");
}
